<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QCentralWidget{
	background-color:black;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="MyTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>50</horstretch>
          <verstretch>7</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabBar::tab{
	width:100px;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #666666, stop: 0.4 #9E9E9E, stop: 1.0 #F9F9F9);
}
QTabBar::tab:hover{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #848484, stop: 0.4 #ADADAD, stop: 1.0 #FFFFFF);
}
QTabBar::tab:selected{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 1 #848484, stop: 0.9 #D3D3D3,stop: 0.2 #9E9E9E, stop: 0 #848484);
}
QTabWidget::pane{
	border: 0 solid white;
	margin: -13px -9px -13px -9px;
}
</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="changeTab">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:#474747;
	border:0px solid blue;
	margin:0px 0px 0px 0px;
	padding: 0px 0px 0px 0px;
}
QPushButton:pressed
{
	border:2px solid #54bae5;
	margin:3px 3px 0px 3px;
}
QPushButton:hover{
	background-color:#504f4f;
	border:2px solid #767676;
}</string>
         </property>
         <attribute name="title">
          <string>Change</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="cursorMode">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/buttons/img/cursor.png</normaloff>:/buttons/img/cursor.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="bridgeMode">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/buttons/img/Bridge.png</normaloff>:/buttons/img/Bridge.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="edgeMode">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/buttons/img/Edge.png</normaloff>:/buttons/img/Edge.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="deleteTab">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:#474747;
	border:0px solid blue;
	margin:0px 0px 0px 0px;
	padding: 0px 0px 0px 0px;
}
QPushButton:pressed
{
	border:2px solid #54bae5;
	margin:3px 3px 0px 3px;
}
QPushButton:hover{
	background-color:#504f4f;
	border:2px solid #ff6868;
}</string>
         </property>
         <attribute name="title">
          <string>Delete</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="bridgeDeleteMode">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/buttons/img/BridgeDelete.png</normaloff>:/buttons/img/BridgeDelete.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="edgeDeleteMode">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/buttons/img/EdgeDelete.png</normaloff>:/buttons/img/EdgeDelete.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="deleteAll">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/buttons/img/Trash.png</normaloff>:/buttons/img/Trash.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>50</height>
             </size>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="functionTab">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:#474747;
	border:0px solid blue;
	margin:0px 0px 0px 0px;
	padding: 0px 0px 0px 0px;
}
QPushButton:pressed
{
	border:2px solid #54bae5;
	margin:3px 3px 0px 3px;
}
QPushButton:hover{
	background-color:#504f4f;
	border:2px solid #e9941a;
}</string>
         </property>
         <attribute name="title">
          <string>Watch</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="visualizeGraph">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/buttons/img/List.png</normaloff>:/buttons/img/List.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="openDfs">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/buttons/img/DFSAlgo.png</normaloff>
              <disabledoff>:/buttons/img/DFSAlgo.png</disabledoff>:/buttons/img/DFSAlgo.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="VisualGraphSceneView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>100</horstretch>
          <verstretch>30</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>325</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VisualGraphSceneView</class>
   <extends>QGraphicsView</extends>
   <header>AppElements/visualgraphsceneview.h</header>
  </customwidget>
  <customwidget>
   <class>MyTabWidget</class>
   <extends>QTabWidget</extends>
   <header>AppElements/mytabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
